controllers/installation_controller.go

```diff
@@ -165,21 +166,26 @@ func (r *InstallationReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	}
 
 	log.V(Log4Debug).Info("Reconciliation complete: A porter agent has been dispatched to apply changes to the installation.")
-	if r.PorterGRPCClient != nil {
-		return r.CheckOrCreateInstallationOutputsCR(ctx, log, inst)
-	}
-	return ctrl.Result{}, nil
+	log.V(Log4Debug).Info(fmt.Sprintf("performing installation outputs for %s", inst.Name))
+	return r.CheckOrCreateInstallationOutputsCR(ctx, log, inst)
 }
 
 func (r *InstallationReconciler) CheckOrCreateInstallationOutputsCR(ctx context.Context, log logr.Logger, inst *v1.Installation) (ctrl.Result, error) {
 	// NOTE: May not want to requeue if this fails
+	porterGRPCClient, conn, err := r.CreateGRPCClient(ctx)
```

Comment:
nit: Guard check in case r.CreateGRPCClient is undefined. Shouldn't hit that though since this is part of the instantiation

Reply:
Guarded against in line 175. Resolving.